src/ValidatorChain.php

```diff
@@ -33,24 +33,17 @@ public function attach($eventName, callable $listener, $priority = 1)
 
     /**
      * Retrieve session storage object
-     *
-     * @return StorageInterface
      */
-    public function getStorage()
+    public function getStorage(): StorageInterface
     {
         return $this->storage;
     }
 
     /**
      * Internal implementation for attaching a listener to the
      * session validator chain.
-     *
-     * @param string   $event
-     * @param callable $callback
-     * @param int      $priority
-     * @return callable
```

Comment:
Can we accept and return a more precise type for `callable` - I don't know what the signature should be but, something like `callable(mixed): bool` perhaps?

Reply:
Actually, nevermind - I can see this extends from `EventManager` so types should be improved there. They are also plain `callable` in `EventManager` so we'll just end up with `MoreSpecificImplementedType` issues…